RxCocoa/Common/DelegateProxyType.swift

```diff
@@ -210,6 +213,14 @@ extension DelegateProxyType {
             proxy.setForwardToDelegate(nil, retainDelegate: retainDelegate)
         }
     }
+    
+    public static func extend<Object: AnyObject>(with factory: @escaping ((Object) -> AnyObject)) {
+        factories.append({ ($0 as? Object).map(factory) })
```

Comment:
Maybe if I follow swift coding guideline, `createProxyForObject(_:)` should be `createProxy(for object:)`.
I will do so once, please let me know if it's wrong.